src/m_config.c

```diff
@@ -824,6 +824,11 @@ static default_t extra_defaults_list[] =
     CONFIG_VARIABLE_INT_HEX(opl_io_port),
 
     //!
+    // OPL chip type.
+	//
```

Comment:
Please don't insert tab characters into files; use spaces. The formatting here is wrong.